cli-plugins/socket/socket.go

```diff
@@ -97,7 +103,7 @@ func (pl *PluginServer) closeAllConns() {
 	pl.mu.Lock()
 	defer pl.mu.Unlock()
 
-	// Prevent new connections from being accepted
+	// Prevent new connections from being accepted.
```

Comment:
FWIW, there's a linter for this; I know some projects I work on that have it enabled, but if we want to enable it, we probably will have to make a big pass on updating many existing GoDocs 🙈 (so I didn't do that yet).